tests/components/zwave_js/test_config_flow.py

```diff
@@ -3880,8 +3880,14 @@ async def mock_backup_nvm_raw():
 
     result = await hass.config_entries.flow.async_configure(result["flow_id"])
 
-    assert result["type"] == FlowResultType.ABORT
-    assert result["reason"] == "restore_failed"
+    assert result["type"] == FlowResultType.FORM
+    assert result["step_id"] == "restore_failed"
+    assert result["description_placeholders"]["file_path"]
+
+    result = await hass.config_entries.flow.async_configure(result["flow_id"], {})
+
+    assert result["type"] == FlowResultType.SHOW_PROGRESS
+    assert result["step_id"] == "restore_nvm"
```

Comment:
Let's continue the flow and let the restore fail again, assert that the restore nvm call count is now 2, and then simulate the user aborting the flow by calling `hass.config_entries.flow.async_abort(result["flow_id"])`, and assert that no flow is in progress anymore.

`assert len(hass.config_entries.flow.async_progress()) == 0`